docker/cygnus-ngsi/Dockerfile

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of fiware-cygnus (FI-WARE project).
+#
+# fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+# General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+# fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+#
+
+FROM centos:6
+MAINTAINER Francisco Romero Bueno <francisco.romerobueno@telefonica.com>
+
+ENV CYGNUS_USER "cygnus"
+ENV CYGNUS_HOME "/opt/fiware-cygnus"
+ENV CYGNUS_VERSION "0.13.0_SNAPSHOT"
+
+ENV GIT_URL_CYGNUS https://github.com/telefonicaid/fiware-cygnus.git
+ENV GIT_REV_CYGNUS develop
```

Comment:
**Note:** in order to have proper tags, you should expand the change to every release branch, as this Dockerfile will always build from `GIT_REV_CYGNUS develop`. See https://github.com/telefonicaid/fiware-orion/issues/1922 to check how they proceed with the same issue :)